backend/tests/Integration/Request/OperationalSummaryTest.php

```
@@ -0,0 +1,368 @@
+use App\Infrastructure\Request\RequestDashboardQuery;
+use Tests\Integration\IntegrationTestCase;
+
+final class OperationalSummaryTest extends IntegrationTestCase
+{
+    public function testOperationalStatusesAndAssignmentsFormExactTotals(): void
+    {
+        $manager = $this->roleUser('summary.manager', 'Руководитель', 'ic_manager');
+        $initiator = $this->createUser('summary.initiator', 'Инициатор');
+        $executor = $this->roleUser('summary.executor', 'Кашин', 'ic_executor');
+
+        $unassigned = $this->request($initiator, 'без исполнителя');
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$unassigned'. (undefined)

(UnusedLocalVariable)